lang/main-enGB.json

```diff
@@ -280,7 +280,23 @@
         "streamKey": "Live stream key",
         "thankYou": "Thank you for using {{appName}}!",
         "token": "token",
-        "tokenAuthFailed": "Sorry, you're not allowed to join this call.",
+        "tokenAuthFailed": "Sorry, you're not allowed to join this call. Possible reasons: {{reason}}",
+        "tokenAuthFailedReason": {
```

Comment:
You don't need to add these, if a translation is missing then it falls back to main.json.

Unrelated: In fact, en-GB is probably dumb to have at this point, since it's the same as the default...

Reply:
I've updated the existing key, tokenAuthFailed, because otherwise it would've omitted the whole thing. Should I do this for all languages?